configs/tasks/rearrange/rearrange_easy_multi_agent.yaml

```diff
@@ -0,0 +1,169 @@
+    - joint
+    - is_holding
+ENVIRONMENT:
+    MAX_EPISODE_STEPS: 1500
+DATASET:
+    TYPE: RearrangeDataset-v0
+    SPLIT: train
+    DATA_PATH: data/datasets/replica_cad/rearrange/v1/{split}/rearrange_easy.json.gz
+    SCENES_DIR: data/replica_cad/
+TASK:
+    TYPE: RearrangeCompositeTask-v0
+    REWARD_MEASURE: move_obj_reward
+    SUCCESS_MEASURE: composite_success
+    SUCCESS_REWARD: 100.0
+    SLACK_REWARD: -0.01
+    END_ON_SUCCESS: true
+    COUNT_OBJ_COLLISIONS: true
+    COUNT_ROBOT_OBJ_COLLS: false
+    CONSTRAINT_VIOLATION_ENDS_EPISODE: false
+    CONSTRAINT_VIOLATION_DROPS_OBJECT: True
+    BASE_ANGLE_NOISE: 0.15
+    BASE_NOISE: 0.05
+    EASY_INIT: false
+    TASK_SPEC_BASE_PATH: configs/tasks/rearrange/pddl/
+    TASK_SPEC: rearrange_easy
+    PDDL_DOMAIN_DEF: replica_cad
+    OBJ_SUCC_THRESH: 0.3
+    ART_SUCC_THRESH: 0.15
+    SENSORS:
+    - RELATIVE_RESTING_POS_SENSOR
+    - TARGET_START_SENSOR
+    - GOAL_SENSOR
+    - JOINT_SENSOR
+    - IS_HOLDING_SENSOR
+    - END_EFFECTOR_SENSOR
+    - TARGET_START_GPS_COMPASS_SENSOR
+    - TARGET_GOAL_GPS_COMPASS_SENSOR
+    FORCE_TERMINATE:
+        TYPE: ForceTerminate
+        MAX_ACCUM_FORCE: 100_000.0
+        MAX_INSTANT_FORCE: 10_000.0
+    MEASUREMENTS:
+    - OBJECT_TO_GOAL_DISTANCE
+    - ROBOT_FORCE
+    - FORCE_TERMINATE
+    - ROBOT_COLLS
+    - COMPOSITE_SUCCESS
+    - NUM_STEPS
+    ACTIONS:
+      AGENT_0_ARM_ACTION:
+          AGENT: 0
+          TYPE: ArmAction
+          ARM_CONTROLLER: ArmRelPosAction
+          GRIP_CONTROLLER: MagicGraspAction
+          ARM_JOINT_DIMENSIONALITY: 7
+          GRASP_THRESH_DIST: 0.15
+          DISABLE_GRIP: false
+          DELTA_POS_LIMIT: 0.0125
+          EE_CTRL_LIM: 0.015
+      AGENT_0_BASE_VELOCITY:
+          AGENT: 0
+          TYPE: BaseVelAction
+          LIN_SPEED: 10.0
+          ANG_SPEED: 10.0
+          ALLOW_DYN_SLIDE: true
+          END_ON_STOP: false
+          ALLOW_BACK: true
+          MIN_ABS_LIN_SPEED: 1.0
+          MIN_ABS_ANG_SPEED: 1.0
+      AGENT_0_PDDL_APPLY_ACTION:
+          AGENT: 0
+          TYPE: PddlApplyAction
+      AGENT_1_ARM_ACTION:
+          AGENT: 1
+          TYPE: ArmAction
+          ARM_CONTROLLER: ArmRelPosAction
+          GRIP_CONTROLLER: MagicGraspAction
+          ARM_JOINT_DIMENSIONALITY: 7
+          GRASP_THRESH_DIST: 0.15
+          DISABLE_GRIP: false
+          DELTA_POS_LIMIT: 0.0125
+          EE_CTRL_LIM: 0.015
+      AGENT_1_BASE_VELOCITY:
+          AGENT: 1
+          TYPE: BaseVelAction
+          LIN_SPEED: 10.0
+          ANG_SPEED: 10.0
+          ALLOW_DYN_SLIDE: true
+          END_ON_STOP: false
+          ALLOW_BACK: true
+          MIN_ABS_LIN_SPEED: 1.0
+          MIN_ABS_ANG_SPEED: 1.0
+      AGENT_1_PDDL_APPLY_ACTION:
+          AGENT: 1
+          TYPE: PddlApplyAction
+      AGENT_0_REARRANGE_STOP:
+          TYPE: RearrangeStopAction
+      AGENT_1_REARRANGE_STOP:
+          TYPE: RearrangeStopAction
+    POSSIBLE_ACTIONS:
```

Comment:
I see, we have action space adopted for multi agent setup. Doest rewards, metrics support multiagent already? Thanks.

Reply:
Yes, it supports metrics but the metrics must be defined per agent. The metrics I include in the default config are agent agnostic.